webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
Take a look at:

https://github.com/TurnerBroadcasting/cnn-hope/blob/develop/webpack.config.js#L71

and

https://github.com/TurnerBroadcasting/cnn-hope/blob/develop/src/components/gallery/main.js

It's worth considering using the handlebars-loader for webpack so that you can keep all your handlebars template  html lgoic for the carousel in a seperate file and pull it in.